styles/filter.css

```diff
@@ -0,0 +1,62 @@
+.filter-container {
+    background-color: var(--tertiary-color);
+    width: 15rem;
+}
+
+.items {
+    padding: 1rem;
+}
+
+.filter {
+    display: flex;
+    justify-content: space-between;
+}
+
+.filter-text {
+    font-size: 1.2rem;
+    font-weight: 700;
+    margin: 1rem 0;
+}
+
+.clear-text {
+    border: none;
+}
+
+.clear-text:hover {
+    color: var(--secondary-color);
+}
+
+.inputs {
+    font-size: 1rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.radio-unchecked {
+    margin: 0.4rem;
+}
+
+.price-range {
+    display: flex;
+    justify-content: space-between;
+    font-size: 1rem;
+}
+
+.slider {
+    width: 100%;
+    height: 2rem;
+    border-radius: 5px;
+    opacity: 0.9;
+    transition: opacity .2s;
+}
+
+.slider::-webkit-slider-thumb {
+    -webkit-appearance: none;
```

Comment:
You don't have to use browser specific (`-webkit-`) styles, that is handled automatically.

Reply:
Ok. Will do it.